src/components/main/HotGiftRankingTag.tsx

```diff
@@ -0,0 +1,71 @@
+import styled from "@emotion/styled";
+
+interface TagProps {
+  isSelected: boolean;
+  onClick: () => void;
+  tagEmoji: string;
+  tagText: string;
+}
+
+const HotGiftTagContainer = styled.button<{ isSelected: boolean }>(
```

Comment:
```suggestion
const HotGiftTagContainer = styled.button(
```

isSelected 안 쓰는 거 같은데.. 요렇게???

Reply:
이 부분은  `<HotGiftTagContainer isSelected={isSelected} onClick={onClick}>`에서 쓰고있습니다!